### PR TITLE
docs: state we support repo perms from multiple code hosts

### PR DESCRIPTION
**Pull request by unknwon**
PS: I linked a blog post, I don't particularly like linking external resources for how to work with our internal system. However, the blog post is very detailed and thus verbose.

### CODE REVIEW

**Review by artemruts — APPROVED** [review]
Nice blogpost 👍

File: doc/admin/repo/permissions.md [hunk 1]
```diff
@@ -4,6 +4,8 @@ Sourcegraph can be configured to enforce repository permissions from code hosts.
 
 Currently, GitHub, GitHub Enterprise, GitLab and Bitbucket Server permissions are supported. Check our [product direction](https://about.sourcegraph.com/direction) for plans to support other code hosts. If your desired code host is not yet on the roadmap, please [open a feature request](https://github.com/sourcegraph/sourcegraph/issues/new?template=feature_request.md).
 
+If the Sourcegraph is configured to sync repositories from multuple code hosts (regardless of if they have the same type, e.g. GitHub + GitHub or GitHub + GitLab), setting up permissions for each of them will make the repoisotry permissions work simultaneously. Setting up a unified SSO for code hosts and Sourcegraph is also possible, read more about how to [Set up Sourcegraph with two GitLab and Keycloak using SAML](https://unknwon.io/posts/200915_setup-sourcegraph-gitlab-keycloak/).
```
Comment (quinnkeast) [inline doc/admin/repo/permissions.md:7]:
```suggestion
If the Sourcegraph instance is configured to sync repositories from multiple code hosts (regardless of whether they are the same code host, e.g. `GitHub + GitHub` or `GitHub + GitLab`), setting up permissions for each will make repository permissions work simultaneously. Setting up a unified SSO for code hosts and Sourcegraph is also possible: how to [Set up Sourcegraph with two GitLab and Keycloak using SAML](https://unknwon.io/posts/200915_setup-sourcegraph-gitlab-keycloak/).
```

Comment (quinnkeast) [reply doc/admin/repo/permissions.md:7]: I've added some tweaks here, but I'm running into a conceptual understanding issue with this part:

>setting up permissions for each will make repository permissions work simultaneously

Can you tell me more about what exactly "...repository permissions work simultaneously" means? I think the term "simultaneously" probably needs to be replaced, but I want to make sure I understand it fully.

Comment (unknwon) [reply doc/admin/repo/permissions.md]: @quinnkeast "simultaneously" here I mean repository permissions work independently for each code host, also from user's POV permissions work as a whole (for all code hosts). Does this help?

Comment (quinnkeast) [reply doc/admin/repo/permissions.md]: I think so! I've updated my suggested change: does this make sense to you?

File: doc/admin/repo/permissions.md [hunk 2]
```diff
@@ -4,6 +4,8 @@ Sourcegraph can be configured to enforce repository permissions from code hosts.
 
 Currently, GitHub, GitHub Enterprise, GitLab and Bitbucket Server permissions are supported. Check our [product direction](https://about.sourcegraph.com/direction) for plans to support other code hosts. If your desired code host is not yet on the roadmap, please [open a feature request](https://github.com/sourcegraph/sourcegraph/issues/new?template=feature_request.md).
 
+If the Sourcegraph instance is configured to sync repositories from multiple code hosts (regardless of whether they are the same code host, e.g. `GitHub + GitHub` or `GitHub + GitLab`), setting up permissions for each will make repository permissions work simultaneously. Setting up a unified SSO for code hosts and Sourcegraph is also possible: how to [Set up Sourcegraph with two GitLab and Keycloak using SAML](https://unknwon.io/posts/200915_setup-sourcegraph-gitlab-keycloak/).
```
Comment (unknwon) [inline doc/admin/repo/permissions.md:7]:
```suggestion
If the Sourcegraph instance is configured to sync repositories from multiple code hosts (regardless of whether they are the same code host, e.g. `GitHub + GitHub` or `GitHub + GitLab`), setting up permissions for each code host will make repository permissions apply holistically on Sourcegraph. 

Setting up a unified SSO for code hosts and Sourcegraph is also possible: how to [Set up Sourcegraph with two GitLab and Keycloak using SAML](https://unknwon.io/posts/200915_setup-sourcegraph-gitlab-keycloak/).
```

File: doc/admin/repo/permissions.md (hunk 1 above)
Comment (unknwon) [reply doc/admin/repo/permissions.md]: Thanks @quinnkeast !

**Pull request merged**